PermissionScope/PermissionScope.swift

```diff
@@ -1003,7 +1075,8 @@ typealias resultsForConfigClosure     = ([PermissionResult]) -> Void
         onCancel = cancelled
         
         DispatchQueue.main.async {
-            while self.waitingForBluetooth || self.waitingForMotion { }
+            while
+                self.waitingForBluetooth || self.waitingForMotion { }
```

Comment:
If I'm understanding it correctly, it will continue looping (and doing nothing) until both the waiting vars are false.